scripts/build_yarpgen.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+if [[ -z "${YARPGEN_HOME}" ]]; then
+    echo "Error! Environmental variable for YARPGEN_HOME is not set!"
+    exit 127
+fi
+
+# Get the source code
+mkdir -p $YARPGEN_HOME
```

Comment:
The source code is already checked-out, so the comment and `mkdir -p` are irrelevant.
```suggestion
```